starknet_devnet/starknet_wrapper.py

```diff
@@ -198,9 +198,10 @@ def get_transaction_status(self, transaction_hash: str):
             if "block_hash" in transaction:
                 ret["block_hash"] = transaction["block_hash"]
 
-            failure_key = "tx_failure_reason"
-            if failure_key in transaction:
-                ret[failure_key] = transaction[failure_key]
+            curr_failure_key = "transaction_failure_reason"
```

Comment:
"transaction_failure_reason" can be defined in constants.py under a name like `FAILURE_REASON_KEY`.